docs/deployment/k8s/index.mdx

```diff
@@ -10,26 +10,30 @@ For high-level discussion of our approach to Kubernetes and compatibility with c
 
 On **this** page, you can find documentation for operators wishing to deploy wasmCloud with Kubernetes, including details on prerequisites such as NATS and how to deploy the operator.
 
+:::info[kubectl support]
+When running wasmCloud on Kubernetes, you can use `kubectl` to deploy [wasmCloud manifests](/docs/ecosystem/wadm/model), check wasmCloud logs, and more. The example below demonstrates standard usage of `kubectl` with wasmCloud.
+:::
+
 ## Getting started with the wasmCloud operator
 
 The wasmCloud operator can be found on GitHub at [https://github.com/wasmCloud/wasmcloud-operator](https://github.com/wasmCloud/wasmcloud-operator). It uses custom resource definitions (CRDs) to define wasmCloud hosts as Kubernetes resources.

```

Comment:
It may be redundant given the `:::info:::` block above, but it might be worth enumerating the 3 things the Operator does for you today:

1. Enables deployment of wasmCloud Hosts into the cluster via Custom Resource (`WasmCloudHostConfig`)
2. Enables interoperability with wadm for deploying wasmcloud applications defined in the [OAM spec](https://oam.dev/) format. 
3. Enables automated creation of Kubernetes Services pointing to wasmcloud applications deployed with a HTTP Server capability (in the future this will become configurable, today it's automatic).